packages/babel-preset-sui/lib/index.js

```diff
@@ -15,33 +44,18 @@ module.exports = {
             'Safari >= 8',
             'iOS >= 8',
             'ie >= 11',
-            'not op_mini all'
+            'not op_mini all',
+            'not dead'
           ]
-        }
+        },
+        useBuiltIns: false
       }
     ],
-    isInstalled(['preact', 'react'], 'babel-preset-react')
-  ]),
-  plugins: [
-    require('babel-plugin-transform-async-generator-functions'),
-    require('babel-plugin-transform-decorators-legacy').default,
-    require('babel-plugin-transform-class-properties'),
-    require('babel-plugin-transform-object-rest-spread'),
-    require('babel-plugin-transform-runtime'),
-    require('babel-plugin-syntax-dynamic-import'),
-    require('babel-plugin-transform-export-extensions'),
-    [
-      require('babel-plugin-transform-react-remove-prop-types').default,
-      {
-        mode: 'wrap'
-      }
-    ]
-  ],
-  env: {
-    development: {
-      plugins: cleanList([
-        isInstalled(['preact', 'react'], 'react-hot-loader/babel')
```

Comment:
We need this for the hot reloading: https://github.com/gaearon/react-hot-loader#getting-started

Reply:
Yikes! I've removed it for testing, I've added a pending task in order to add it. Good catch!